api/helpers/aggregators.js

```diff
@@ -322,12 +322,14 @@ const getConvertedValue = (item, entry) => {
       // Bool
       const tempObj = {};
       tempObj[item] = true;
-      tempObj.active = true;
       return tempObj;
     } else if (entry === 'false') {
       console.log("bool");
       // Bool
-      return { [item]: false };
+      console.log("bool");
```

Comment:
Can we get right of the console logs while we are in here? Also, should we account for 'False' and 'FALSE' (or any other variation)?

Reply:
Yeah sounds good, I just kept the original code as is. Change incoming

Reply:
Just handling toLowerCase for "true" and "false", in the event someone needs other values for another valid query.